layers/poi/class.sql

```diff
@@ -46,24 +48,28 @@ RETURNS TEXT AS $$
         WHEN subclass IN ('books','library') THEN 'library'
         WHEN subclass IN ('university','college') THEN 'college'
         WHEN subclass IN ('hotel','motel','bed_and_breakfast','guest_house','hostel','chalet','alpine_hut','camp_site') THEN 'lodging'
-        WHEN subclass IN ('chocolate','confectionery') THEN 'ice_cream'
+        WHEN subclass IN ('chocolate','confectionery', 'ice_cream') THEN 'ice_cream'
         WHEN subclass IN ('post_box','post_office') THEN 'post'
-        WHEN subclass IN ('cafe') THEN 'cafe'
+        WHEN subclass IN ('cafe', 'coffee') THEN 'cafe'
```

Comment:
amenity=cafe and shop=coffee are very different kind of POIs, we should not group them.